source/Interfaces/NetWorth/server/servermain.cpp

```diff
@@ -0,0 +1,64 @@
+/**
+ * This file is part of the Fall 2023, CSE 491 course project.
+ * @brief A simplistic main file to demonstrate a system.
+ * @note Status: PROPOSAL
+ **/
+
+// Include the modules that we will be using.
+#include <sstream>
+#include<string>
+#include<vector>
+#include "../../../core/Data.hpp"
+#include "../../TrashInterface.hpp"
+#include "../../../Agents/PacingAgent.hpp"
+#include "../../../Worlds/MazeWorld.hpp"
+#include "../NetworkInterface.hpp"
+#include "networkingworld.hpp"
+
+int main()
+{
+    //Create the world on the server side upon initialization
+    //Add the pacing agents to just simply walk around
+    //Once everything is added we simply wait for a response from a client that is connecting.
+    std::shared_ptr<cse491::netWorth::NetworkMazeWorld> world = std::make_shared<cse491::netWorth::NetworkMazeWorld>();
+    world->AddAgent<cse491::PacingAgent>("Pacer 1").SetPosition(3,1);
+    world->AddAgent<cse491::PacingAgent>("Pacer 2").SetPosition(6,1);
+    world->AddAgent<cse491::TrashInterface>("Interface").SetProperty("symbol", 'Z');
+
+    std::shared_ptr<cse491::netWorth::ServerInterface> serverInterface = std::make_shared<cse491::netWorth::ServerInterface>();
+
+    world->SetServer(serverInterface);
+
+    UdpSocket * serverSocket = serverInterface->GetSocket();
+
+    std::cout << "Server IP Address: " << sf::IpAddress::getLocalAddress() << std::endl;
+
+    //Establish an initial connection
```

Comment:
Can establishing the connection for in the network interface constructor?  Then it should be possible to set it up with multiple interfaces without them stepping on each other's toes (I think...)  It will also keep everything more modular.